content/docs/reference-glossary.md

```diff
@@ -1,43 +1,43 @@
-## ES6, ES2015, ES2016, etc {#es6-es2015-es2016-etc}
+## ES6, ES2015, ES2016, आदि {#es6-es2015-es2016-etc}
 
-These acronyms all refer to the most recent versions of the ECMAScript Language Specification standard, which the JavaScript language is an implementation of. The ES6 version (also known as ES2015) includes many additions to the previous versions such as: arrow functions, classes, template literals, `let` and `const` statements. You can learn more about specific versions [here](https://en.wikipedia.org/wiki/ECMAScript#Versions).
+ये सभी शब्द ECMAScript लैंग्वेज स्पेसिफिकेशन स्टैंडर्ड, जो जावास्क्रिप्ट भाषा का एक कार्यान्वयन है, के सबसे हाल के वर्शन्स को संदर्भित करते हैं। ES6 वर्शन (जिसे ES2015 के रूप में भी जाना जाता है) में पिछले वर्शन्स के मुकाबले में कई परिवर्धन शामिल किये गए हैं जैसे की: एरो फ़ंक्शंस, क्लासेस, टेम्प्लेट लिटेरल्स, `let` और `const` स्टेटमेंट। आप [यहां](https://en.wikipedia.org/wiki/ECMAScript#Versions) विशिष्ट वर्शन्स के बारे में अधिक जान सकते हैं।
 
-## Compilers {#compilers}
+## कम्पाइलर्स {#compilers}
 
-A JavaScript compiler takes JavaScript code, transforms it and returns JavaScript code in a different format. The most common use case is to take ES6 syntax and transform it into syntax that older browsers are capable of interpreting. [Babel](https://babeljs.io/) is the compiler most commonly used with React.
+एक जावास्क्रिप्ट कंपाइलर जावास्क्रिप्ट कोड लेता है, और उसे बदल कर एक अलग प्रारूप में जावास्क्रिप्ट कोड देता है। सबसे आम उपयोग ES6 सिंटैक्स लेने और इसे ऊस सिंटैक्स में बदलने का है जिसे कि पुराने ब्राउज़र व्याख्या करने में सक्षम हैं। आमतौर पर React के साथ [Babel](https://babeljs.io/) कंपाइलर का उपयोग किया जाता है।
 
-## Bundlers {#bundlers}
+## बंडलर्स {#bundlers}
 
-Bundlers take JavaScript and CSS code written as separate modules (often hundreds of them), and combine them together into a few files better optimized for the browsers. Some bundlers commonly used in React applications include [Webpack](https://webpack.js.org/) and [Browserify](http://browserify.org/).
+बंडलर्स अलग-अलग मॉड्यूल के रूप में लिखे गए(अक्सर सैकड़ों में) जावास्क्रिप्ट और css कोड को एक साथ जोड़कर कुछ फाइल्स बनाते है जो की ब्राउज़रों के लिए बेहतर अनुकूलित हैं। आमतौर पर React एप्लिकेशन में उपयोग किए जाने वाले कुछ बंडलों में [Webpack](https://webpack.js.org/) और [Browserify](http://browserify.org/) शामिल हैं।
 
-## Package Managers {#package-managers}
+## Package मैनेजर्स {#package-managers}
 
-Package managers are tools that allow you to manage dependencies in your project. [npm](https://www.npmjs.com/) and [Yarn](https://yarnpkg.com/) are two package managers commonly used in React applications. Both of them are clients for the same npm package registry.
+Package मैनेजर वे उपकरण हैं जो आपको अपनी परियोजना में निर्भरता का प्रबंधन करने की अनुमति देते हैं। [npm](https://www.npmjs.com/) और [Yarn](https://yarnpkg.com/) दो package मैनेजर हैं जो आमतौर पर React एप्लिकेशन में उपयोग किए जाते हैं। ये दोनों समान npm package रजिस्ट्री के लिए क्लाइंट हैं।
```

Comment:
Package मैनेजर वे उपकरण हैं जो आपको अपनी परियोजना में निर्भरता का प्रबंधन करने की अनुमति देते हैं। too complex

Reply:
ये दोनों समान npm package रजिस्ट्री के लिए क्लाइंट  -> liye is unnecessary

Reply:
> Package मैनेजर वे उपकरण हैं जो आपको अपनी परियोजना में निर्भरता का प्रबंधन करने की अनुमति देते हैं। too complex

Package मैनेजर वे उपकरण हैं जो आपको अपने प्रोजेक्ट में डेपेंडेंसीएस का प्रबंधन करने की अनुमति देते हैं।